Automattic/jetpack
# Dash: allow turning on searchable modules again

#### Changes proposed in this Pull Request:

* ﻿This bug was introduced in #18837, and stopped one from enabling searchable modules from the Jetpack dashboard, because the logic was not looking for the right module data.

**Before**

- When the module is active
![image](https://user-images.githubusercontent.com/426388/145567921-fcdb8869-dce6-4979-aa67-da1e5c42e6c0.png)


**After**

- When the module is active
![image](https://user-images.githubusercontent.com/426388/145567853-c64b979b-c8f1-48b3-8347-4b42009a86f7.png)

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start from a site that's not connected to WordPress.com.
* Start the connection process, but do not authenticate / connect with your WordPress.com user; stop the WordPress.com connection process then and come back to the Jetpack dashboard in wp-admin.
* Go to Jetpack > Modules (the old module screen, accessible via the link in the footer)
* Ensure that the contact form, enhanced distribution, and notifications features are inactive.
* Go to Jetpack > Settings, and search for each one of those features.
* For the feature that relies on a user connection to WordPress.com, you should see an invitation to connect your user.
* For the others, you should see a prompt to activate the feature (this did not work anymore before this patch).
* Connect your user to WordPress.com.
* Go back to Jetpack > Settings, and search for the features again; you should see prompts to enable the features for each feature.
* Enable the features.
* Refresh the page, and search for the features again; you should see a card explaining what the feature does.
